frontend/src/pages/login/LoginPage.tsx

```diff
@@ -0,0 +1,126 @@
+
+import React, { useState } from 'react';
+import { Container, Grid, TextField, Button, Checkbox, FormControlLabel, Typography, Link } from '@mui/material';
+import logo from '../../assets/images/Logo for auth.jpg';
+import image from '../../assets/images/Stuck at Home Imagination.jpg';
+import './LoginPage.css';
+
+
+const LoginPage: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [rememberMe, setRememberMe] = useState(false);
+  
+
+  const handleLogin = () => {
+    console.log('Email:', email);
+    console.log('Password:', password);
+    console.log('Remember Me:', rememberMe);
+  };
+
+  return (
+    <Container>
+      
+        <img src={logo} alt="Image" style={{ width: '140px', height: '30px', top: '40px', left:'76px' }} />
+      
+      <Grid container justifyContent="center" alignItems="center" style={{ height: '100vh' }}>
+        <Grid item xs={12} sm={6}>
+          
+            <Typography style={{
+              
+              font: 'Poppins',
```

Comment:
The code in this page is very poorly structured, I will try making an automated tool for that, but for now, please format it properly

Reply:
My prettier formatter didn't work. I fix it now.